sql-plugin/src/main/spark320/scala/org/apache/spark/sql/errors/ConvUtils.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*** spark-rapids-shim-json-lines
+{"spark": "320"}
+{"spark": "321"}
+{"spark": "321cdh"}
+{"spark": "322"}
+{"spark": "323"}
+{"spark": "324"}
+{"spark": "330"}
+{"spark": "330cdh"}
+{"spark": "330db"}
+{"spark": "331"}
+{"spark": "332"}
+{"spark": "332cdh"}
+{"spark": "332db"}
+{"spark": "333"}
+{"spark": "334"}
+{"spark": "340"}
+{"spark": "341"}
+{"spark": "341db"}
+{"spark": "342"}
+{"spark": "343"}
+{"spark": "344"}
+spark-rapids-shim-json-lines ***/
+
+package org.apache.spark.sql.errors
+
+object ConvUtils {
+  // only Spark versions >= 350 support this function
+  def overflowInConvError(): Unit = throw new UnsupportedOperationException()
```

Comment:
Personally shims for this are not needed. Since there are already shims for anisMode to take care of the difference.

Reply:
Spark 350 supports this function `overflowInConvError` 
Refer to [link](https://github.com/apache/spark/blob/v3.5.0/sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala#L307) 

Spark 330 does not support this function, refer to [link](https://github.com/apache/spark/blob/v3.3.0/sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala)

So we need to add shim for this.

BTW,  I found a bug, Spark 340 also supports  `overflowInConvError`, will update.

Reply:
OK, the implemenetation should be different.